libs/common/src/components/screens/receive/index.tsx

```diff
@@ -89,7 +89,13 @@ export const ReceiveScreen = observer(function ReceiveScreen() {
             paddingVertical: 20,
             paddingHorizontal: 30,
           }}
-          onPress={() => onShare(address)}
+          onPress={() => {
+            if (Platform.OS === "web") {
+              navigator.clipboard.writeText(address);
+            } else {
+              onShare(address);
+            }
+          }}
```

Comment:
This is not chaning anything. `onShare` is already doing that. The problem is related to iframe security policies.